server/src/main/java/com/thoughtworks/go/server/dashboard/GoDashboardCache.java

```diff
@@ -57,12 +53,16 @@ public void replaceAllEntriesInCacheWith(List<GoDashboardPipeline> newPipelinesT
         cacheHasChanged();
     }
 
-    public GoDashboardPipelines allEntriesInOrder() {
+    public GoDashboardPipelines allEntries() {
         return dashboardPipelines;
     }
 
+    public List<GoDashboardPipeline> allEntriesInOrder() {
```

Comment:
I would remove this method and all the corresponding tests. If there is no need for it, why keep it?